src/components/tabs/dataset-tab.tsx

```diff
@@ -79,6 +96,13 @@ export const DatasetTab: React.FC = () => {
   return (
     <div className="App">
       <h1>NASA Earth Observatory</h1>
+      <DatasetSelector selectedDataset={selectedDatasetId} onDatasetChange={handleDatasetChange} />
+      {selectedDataset && (
+        <div className="dataset-info">
+          <h2>{selectedDataset.label}</h2>
+          <img src={selectedDataset.legendImage} alt={selectedDataset.label} />
```

Comment:
I don't have a link to the design spec, so I'm not sure what this is supposed to look like, but if you centered this it would be a little effort to make it look much better.

Reply:
You can get a link to the design here: https://concord-consortium.atlassian.net/browse/NASAEARTH-1
The datasets are left aligned in the design. So it is probably better to keep them that way and then when the rest of stuff in the plugin is removed they'll look better.